.gitignore

```diff
@@ -397,11 +400,81 @@ FodyWeavers.xsd
 # JetBrains Rider
 *.sln.iml
 
-# Credentials for the DSharpPlus.Test bot
-DSharpPlus.Test/config.json
```

Comment:
this is dangerous. we absolutely should not do this.

Reply:
Honestly I think we should just create a better test bot. But yeah I'll readd that